src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/rules-list/rules-list.component.ts

```diff
@@ -33,12 +41,28 @@ export class RulesListComponent extends PrometheusListHelper implements OnInit {
   ngOnInit() {
     super.ngOnInit();
     this.columns = [
-      { prop: 'name', name: $localize`Name` },
-      { prop: 'labels.severity', name: $localize`Severity` },
-      { prop: 'group', name: $localize`Group` },
-      { prop: 'duration', name: $localize`Duration`, pipe: new DurationPipe() },
-      { prop: 'query', name: $localize`Query`, isHidden: true },
-      { prop: 'annotations.description', name: $localize`Description` }
+      { prop: 'name', name: $localize`Name`, flexGrow: 2 },
+      {
+        prop: 'labels.severity',
+        name: $localize`Severity`,
+        flexGrow: 1,
+        cellTemplate: this.severityTpl
+      },
+      { prop: 'group', name: $localize`Group`, flexGrow: 1, cellTemplate: this.groupTpl },
+      { prop: 'duration', name: $localize`Duration`, pipe: new DurationPipe(), flexGrow: 1 },
+      { prop: 'query', name: $localize`Query`, isHidden: true, flexGrow: 1 },
+      { prop: 'annotations.description', name: $localize`Description`, flexGrow: 6 }
```

Comment:
What do you think about replacing this description with the summary? 

![CPT2206241646-1269x215](https://user-images.githubusercontent.com/37327689/175560310-9ca6de62-7959-45eb-9d56-dc829245ab19.png)

BTW, given that this tab (Alerts) is a super-set of the Active Alerts, would it make sense to merge both tabs in a single one, and add a filter (e.g.: [Patternfly "toggle group"](https://www.patternfly.org/v4/guidelines/filters/#toggle-group)) to the table to display only active or all? Like in the Prometheus UI?

![image](https://user-images.githubusercontent.com/37327689/175560684-eb445cd5-7340-45e4-b7fe-fe090c026eb2.png)

Reply:
> What do you think about replacing this description with the summary?
yes, makes sense, I have updated the screenshot with this change.
> 
> ![CPT2206241646-1269x215](https://user-images.githubusercontent.com/37327689/175560310-9ca6de62-7959-45eb-9d56-dc829245ab19.png)
> 
> BTW, given that this tab (Alerts) is a super-set of the Active Alerts, would it make sense to merge both tabs in a single one, and add a filter (e.g.: [Patternfly "toggle group"](https://www.patternfly.org/v4/guidelines/filters/#toggle-group)) to the table to display only active or all? Like in the Prometheus UI?
Yes, I agree here as well..we can take this up in a follow up PR.
> 
> ![image](https://user-images.githubusercontent.com/37327689/175560684-eb445cd5-7340-45e4-b7fe-fe090c026eb2.png)